src/main/resources/assets/immersiveengineering/lang/en_us.json

```diff
@@ -731,15 +731,19 @@
   "effect.immersiveengineering.concrete_feet": "Concrete Feet",
   "effect.immersiveengineering.flashed": "Flashed",
   "manual.immersiveengineering.manual": "Engineer's Manual",
-  "manual.immersiveengineering.update": "UPDATE NEWS!",
-  "manual.immersiveengineering.general": "Overview and Resources",
-  "manual.immersiveengineering.early_machines": "Workbenches & Furnaces",
+  "manual.immersiveengineering.updates_and_introduction": "Updates & Introduction!!",
+  "manual.immersiveengineering.introduction": "Introduction",
+  "manual.immersiveengineering.other_engineers": "Other Engineers",
+  "manual.immersiveengineering.resources": "Resources",
+  "manual.immersiveengineering.storage_transport": "Storage & Transport",
   "manual.immersiveengineering.construction": "Construction",
-  "manual.immersiveengineering.energy": "Power, wires, generators",
-  "manual.immersiveengineering.test": "A subcategory!?",
+  "manual.immersiveengineering.electrical_grids": "Electrical Grids",
+  "manual.immersiveengineering.explosives_weaponry": "Explosives & Weaponry",
   "manual.immersiveengineering.tools": "Tools",
-  "manual.immersiveengineering.machines": "Simple Machines",
-  "manual.immersiveengineering.heavymachines": "Heavy Machinery",
+  "manual.immersiveengineering.workbenches_machinery": "Workbenches & Machinery",
+  "manual.immersiveengineering.multiblock_constructions": "Multiblock Constructions",
+  "manual.immersiveengineering.heavy_machinery": "Heavy Machinery",
+  "manual.immersiveengineering.test": "A subcategory!?",
```

Comment:
...Why do we have this in here still? xD

Reply:
I 'unno, I left it. I'll remove it though.